Makefile

```diff
@@ -169,15 +174,21 @@ help:
 	@echo '  uarch-tests                - build and run microarchitecture rv64i instruction tests'
 	@echo '  uarch-tests-with-linux-env - build and run microarchitecture rv64i instruction tests using the linux-env docker image'
 
+checksum:
+	@cd $(DEPDIR) && sha1sum -c shasumfile
```

Comment:
sha1sum -> shasum

Reply:
I intentionally changed this, so we don't have to add `perl` when packaging for Linux distributions (which is odd), while `sha1sum` comes with `coreutils` package in Linux distributions and is more widely available.

Any motivation to keep using `shasum` ?